www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,83 @@
+var React = require('react/addons');
+var classNames = require('classnames');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
+  render: function () {
+    return (<div className="option-panel">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Checkbox = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      linkState: this.linkState,
+      checkedLabel: true,
+      uncheckedLabel: false
+    };
+  },
+  onChange: function (e) {
+    var val = e.target.checked ? this.props.checkedLabel : this.props.uncheckedLabel;
+    this.valueLink.requestChange(val);
+  },
+  isChecked: function () {
+    return this.valueLink.value === this.props.checkedLabel;
+  },
+  render: function () {
+    var linkState = this.props.linkState;
+    this.valueLink = linkState(this.props.id);
+    return (<label className={classNames('label', {selected: this.isChecked()})}>
+      <input className="sr-only" checked={this.isChecked()} onChange={this.onChange} type="checkbox" />
+      <img className="icon" src={this.props.icon}/>
+    </label>);
+  }
+});
+
+var CheckboxSet = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    linkState: this.linkState
+  },
+  render: function () {
+    return (<OptionPanel>
+      {this.props.options.map((props, i) => <Checkbox {...props} key={i} linkState={this.props.linkState} />)}
```

Comment:
forgot to flag this earlier - I doubt it's extremely important since we're using pretty much "fixed lists" here, but you generally want a key that identifies the thing you're building, so in this cause that would be the option value, not its index (if the array is reordered but we use array index as key, React will diff everything, instead of going "these elements are simply reordered")